momoIOS/Authentication/Controller/InputMemberInfoController.swift

```diff
@@ -122,50 +43,148 @@ class InputMemberInfoController: UIViewController {
+            designerButton.backgroundColor = .rgba(237, 234, 255, 1)
+            designerButton.layer.borderColor = UIColor.main.cgColor
+            designerButton.layer.borderWidth = 2
+            designerButton.jobLabel.textColor = .rgba(132, 112, 255, 1)
+            
+            developerButton.backgroundColor = .white
+            developerButton.layer.borderColor = UIColor.pastbox.cgColor
+            developerButton.layer.borderWidth = 1
+            developerButton.jobLabel.textColor = .gray500
         } else { // developer
-            designerButton.setTitleColor(.gray, for: .normal)
-            designerButton.layer.borderColor = UIColor.gray.cgColor
-            developerButton.setTitleColor(.black, for: .normal)
-            developerButton.layer.borderColor = UIColor.black.cgColor
+            selectedButtonTag = 2
+            developerButton.backgroundColor = .rgba(237, 234, 255, 1)
+            developerButton.layer.borderColor = UIColor.main.cgColor
+            developerButton.layer.borderWidth = 2
+            developerButton.jobLabel.textColor = .rgba(132, 112, 255, 1)
+            
+            designerButton.backgroundColor = .white
+            designerButton.layer.borderColor = UIColor.pastbox.cgColor
+            designerButton.layer.borderWidth = 1
+            designerButton.jobLabel.textColor = .gray500
         }
     }
     
     // MARK: - Helpers
     
     private func setupCustomNav() {
+        let navBar = self.navigationController?.navigationBar
+        navBar?.isHidden = false
+        navBar?.tintColor = .black
+        navBar?.backIndicatorImage = UIImage(systemName: "arrow.left")
+        navBar?.backIndicatorTransitionMaskImage = UIImage(systemName: "arrow.left")
+        navBar?.topItem?.title = ""
+        
         let title = UILabel()
         title.text = "회원정보입력"
         self.navigationItem.titleView = title
-        self.navigationItem.titleView?.tintColor = .black
     }
     
-    private func setupLayout() {
-        let inputViews = [setInputNameView(), setInputYearView(), setInputJobView()]
-        inputViews.forEach { view in
-            view.snp.makeConstraints { make in
-                make.height.greaterThanOrEqualTo(100)
-            }
+    private func setupJobButtonView() {
+        jobButtonView.addArrangedSubviews(designerButton, developerButton)
+        jobButtonView.axis = .horizontal
+        jobButtonView.spacing = 15
+        
+        let designerTapGesture = UITapGestureRecognizer(target: self, action: #selector(handleJobSelection(sender:)))
+        designerButton.addGestureRecognizer(designerTapGesture)
+        let developerTapGesture = UITapGestureRecognizer(target: self, action: #selector(handleJobSelection(sender:)))
+        developerButton.addGestureRecognizer(developerTapGesture)
+        
+        designerButton.snp.makeConstraints { make in
+            make.height.equalTo(180)
+            make.width.equalTo((view.frame.width - 63) / 2)
         }
```

Comment:
스택뷰에서 빼서 사용하면 안줄어들거에용
```suggestion
        designerButton.snp.makeConstraints { make in
            make.top.equalTo(jobLabel.snp.bottom).offset(14)
            make.leading.equalToSuperview().inset(24)
            make.height.equalTo(180)
        }
        
        developerButton.snp.makeConstraints { make in
            make.top.size.equalTo(designerButton)
            make.leading.equalTo(designerButton.snp.trailing).inset(15)
            make.trailing.equalToSuperview().inset(24)
        }
```

참고: https://github.com/Nexters/momo-iOS/blob/main/momoIOS/Common/UI/CommonBottomAlert.swift
139-150 라인 `cancelButton`, `confirmButton`

Reply:
또 하나 배웁니다 .. 🫶